octoprint_mrbeam/iobeam/lid_handler.py

```diff
@@ -724,7 +728,7 @@ def serve_pictures(self, cam, pic_settings=None, cam_params=None, out_pic_size=N
 				# Just tell front end that there was an error
 				self._send_frontend_picture_metadata(correction_result)
 
-			analytics = dict_merge(analytics, dict(
+			self.analytics = dict_merge(analytics, dict(
```

Comment:
analytics now become a class attribut in order to be fetched when we upload a raw image asynchronously to this function.